scripts/unity-utils.ps1

```diff
@@ -60,9 +60,14 @@ function RunUnity([string] $unityPath, [string[]] $arguments, [switch] $ReturnLo
         {
             Write-Host "::endgroup::"
         }
-        if ($stdout -match "No valid Unity Editor license found. Please activate your license.")
+        
+        $hasLicenseError = $stdout -match "No valid Unity Editor license found. Please activate your license."
+        # In Unity 6.0, building on mobile with no license available errors with "unsuppored". Retrying works :)
+        $hasUnsupportedTargetError = $stdout -match "Error building player because build target was unsupported"
+        
+        if ($hasLicenseError -or $hasUnsupportedTargetError)
         {
-            $msg = "Unity failed because it couldn't acquire a license."
+            $msg = if ($hasLicenseError) { "Unity failed because it couldn't acquire a license." } else { "Unity failed because build target was unsupported." }
```

Comment:
```suggestion
            $msg = if ($hasLicenseError) { "Unity failed because it couldn't acquire a license." } else { "Unity failed because build target was unsupported but it's probably a license issue." }
```